fe/fe-core/src/main/java/com/starrocks/backup/BackupJob.java

```diff
@@ -358,6 +358,21 @@ public synchronized void run() {
         if (!status.ok() && state != BackupJobState.UPLOAD_INFO) {
             cancelInternal();
         }
+
+        // just use for FEUT, clear the path
+        if (testPrimaryKey && state == BackupJobState.FINISHED && localJobDirPath != null) {
```

Comment:
I don't think it is appropriate to change the original code just for UT.
Why not delete this path in the UT? That seems a better way than this.

Reply:
done, has fixed the code